Windows/MainWindow.UIControls.cs

```diff
@@ -168,6 +168,7 @@ public partial class MainWindow : Window {
             // update the name of the map in recently opened folders
             recentMaps.RemoveRecentlyOpened(beatMap.GetPath());
             recentMaps.AddRecentlyOpened((string)beatMap.GetValue("_songName"), beatMap.GetPath());
+            recentMaps.Write();
```

Comment:
@PKBeam 
Would it make more sense to do it in TxtSongName_LostFocus to avoid writing to the filesystem too many times?

Reply:
I don't think this really makes any difference - a couple of extra kilobytes' written data will go unnoticed. It's fine to leave it here.